executorlib/standalone/cache/spawner.py

```
@@ -0,0 +1,39 @@
+import subprocess
+import time
+from typing import Optional
+
+
+def execute_in_subprocess(
+    command: list,
+    task_dependent_lst: list = [],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)